packages/material-ui-lab/src/TimelineOppositeContent/TimelineOppositeContent.test.js

```diff
@@ -21,13 +22,71 @@ describe('<TimelineOppositeContent />', () => {
     skip: ['componentProp', 'componentsProp', 'themeVariants'],
   }));
 
-  it('when align right should have alignRight class', () => {
+  it('should have alignLeft class when inside of a left-positioned timeline', () => {
     const { getByText } = render(
-      <Timeline align="right">
+      <Timeline position="left">
         <TimelineOppositeContent>content</TimelineOppositeContent>
       </Timeline>,
     );
 
+    expect(getByText('content')).to.have.class(classes.alignLeft);
+  });
+
+  it('should have alignRight class when inside of a right-positioned timeline', () => {
+    const { getByText } = render(
+      <Timeline position="right">
+        <TimelineOppositeContent>content</TimelineOppositeContent>
+      </Timeline>,
+    );
+
+    expect(getByText('content')).to.have.class(classes.alignRight);
+  });
+
+  it('should have alignLeft class when inside of a left-positioned timeline and a left-aligned item', () => {
+    const { getByText } = render(
+      <Timeline position="left">
+        <TimelineItem align="left">
+          <TimelineOppositeContent>content</TimelineOppositeContent>
+        </TimelineItem>
+      </Timeline>,
+    );
+
+    expect(getByText('content')).to.have.class(classes.alignLeft);
+  });
+
+  it('should have alignLeft class when inside of a right-positioned timeline and a left-aligned item', () => {
+    const { getByText } = render(
+      <Timeline position="right">
+        <TimelineItem align="left">
+          <TimelineOppositeContent>content</TimelineOppositeContent>
+        </TimelineItem>
+      </Timeline>,
+    );
+
+    expect(getByText('content')).to.have.class(classes.alignLeft);
+  });
+
+  it('should have alignRight class when inside of a left-positioned timeline and a right-aligned item', () => {
+    const { getByText } = render(
+      <Timeline position="left">
+        <TimelineItem align="right">
+          <TimelineOppositeContent>content</TimelineOppositeContent>
+        </TimelineItem>
+      </Timeline>,
+    );
+
+    expect(getByText('content')).to.have.class(classes.alignRight);
+  });
+
+  it('should have alignRight class when inside of a right-positioned timeline and a right-aligned item', () => {
+    const { getByText } = render(
+      <Timeline position="right">
+        <TimelineItem align="right">
```

Comment:
Shouldn't it be?

```suggestion
      <Timeline position="right">
        <TimelineItem position="left">
```

Reply:
I think I misunderstood, should we rename the `align` property to `position`? Now `position` only exists on the `<Timeline />` component, but I think it's just a naming misunderstanding

Reply:
I'd say it should be `position` on both components.

Reply:
Done, I have replaced `align` everywhere. I don't know how to handle the CI failure, it seems related to `<DesktopDatePicker />`